doc/rm/comportability_specification/index.md

```diff
@@ -440,18 +440,70 @@ Unlike interrupts, there is no software component to alerts at the peripheral, t
 Each peripheral can send zero or more alerts, where each is a distinguishable security threat.
 Each alert originates in some internal event, and must be specially handled within the peripheral, and then within the alert handler module.
 
+Alerts of comportable IPs in the system must be in either of the following two categories:
+
+1. *Recoverable*, one-time triggered alerts.
+This category is for regular alerts that are due to recoverable error conditions.
+The alert sender transmits one single alert event when the corresponding error condition is asserted.
+
+2. *Fatal* alerts that are continuously triggered until reset.
+This category is for highly critical alerts that are due to terminal error conditions.
+The alert sender continuously transmits alert events until the system is reset.
+
+It is recommended that fatal alerts also trigger local security countermeasures, if they exist.
+For example, a redundantly encoded FSM that is glitched into an invalid state is typically considered to be a fatal error condition.
+In this case, a local countermeasure could be to move the FSM into a terminal error state in order to render the FSM inoperable until the next reset.
+
+The table below lists a few common error conditions and the recommended alert type for each of those errors.
+
+Error Event                                                | Regular IRQ | Recoverable Alert | Fatal Alert
+-----------------------------------------------------------|-------------|-------------------|-------------
+ECC correctable in NVM (OTP, Flash)                        | x           | x                 |
+ECC uncorrectable in Flash                                 | x           | x                 |
+ECC uncorrectable in OTP                                   | (x)         |                   | x
+Any ECC / Parity error in SRAMs or register files          | (x)         |                   | x
+Glitch detectors (e.g., invalid FSM encoding)              | (x)         |                   | x
+Incorrect usage of security IP (e.g., AES config register) | x           | x                 |
+Incorrect usage of regular IP                              | x           |                   |
+
+(x): optional
+
+The column "Regular IRQ" indicates whether the corresponding error condition should also send out a regular IRQ.
+Fatal alerts are not required to send out regular IRQs, since they cannot be recovered via SW actions.
+
+### Defining Alerts
+
+The Hjson configuration file defined above specifies all that needs to be known about the alerts in the standard case.
+The following sections specify what comes out of various tools based upon the simple list defined in the above example.
+
+In terms of naming convention, recoverable alerts must be prefixed with `recov_*`, whereas fatal alerts must be prefixed with `fatal_*`.
+If a module only outputs one alert of each type, it is permissible to name the alerts `recoverable`/`recov` and/or `fatal`.
```

Comment:
nit: To make these in line with interrupts (prefixed with `intr_`), should we prefix with `alert_`, example: `alert_recoverable_` and `alert_fatal_` to make it clear these are alerts? Also, it will be better to stick to one convention and always name the alert accordingly based on what it represents, rather than also allow plain `recov` and `fatal` to be listed as alert signals.

Reply:
Since these names live in a namespace that contains just alerts, I'd prefer not adding an `alert_` prefix: it conveys no information.

Reply:
I agree with Sri in that we don't need choices in naming -- just pick "recov" or "recoverable" and we can update existing IP to use it.

Reply:
I thought having the `alert_` prefix might make it easy to grep those signals in the RTL. and follow them during debug. Not a big deal though..

Reply:
I believe we do not require the interrupts to be prefixed with `intr_` in the Hjson, since the topgen/reggen scripts prepend that automatically when stitching together the toplevel / generating the auto interrupt registers. E.g., when the interrupt name is `foo` in the Hjson, the RTL port name will be `intr_foo_o`.

Hence, adding this prefix in Hjson is not required, since the prefix can be inferred automatically.

Reply:
(That being said, note that alerts are currently wired up a bit differently than interrupts anyways. 
I.e., we do not expose them as separate unique signals in the port list, but rather just expose one parametric packed array with a generic name.
We can align how IRQs and alerts are exposed at the module interfaces if we think that would be cleaner - but this is a separate operation.)

Reply:
I agree regarding the `fatal_` / `recov_` prefixes and alert naming... I agree that we should only allow for one convention and stick to that. 
Let me amend the wording accordingly.

Reply:
Updated.